kong/api/crud_helpers.lua

```diff
@@ -5,16 +5,43 @@ local app_helpers = require "lapis.application"
 
 local _M = {}
 
+--- Will look up a value in the dao.
+-- Either by `id` field or by the field named by 'alternate_field'. If the value is NOT
+-- a uuid, then by the 'alternate_field'. If it is a uuid then it will first try
```

Comment:
uuid's have a specific format and can be validated to conform to that format. Though our check is not doing that because in the past we created bad uuids.